tool/gravity/cli/checks.go

```diff
@@ -17,28 +17,46 @@ limitations under the License.
 package cli
 
 import (
+	"context"
 	"fmt"
 	"io/ioutil"
 
+	"github.com/gravitational/gravity/lib/app"
 	"github.com/gravitational/gravity/lib/checks"
+	"github.com/gravitational/gravity/lib/constants"
+	"github.com/gravitational/gravity/lib/fsm"
 	"github.com/gravitational/gravity/lib/localenv"
+	"github.com/gravitational/gravity/lib/ops"
+	"github.com/gravitational/gravity/lib/pack"
 	"github.com/gravitational/gravity/lib/schema"
 
+	"github.com/fatih/color"
 	pb "github.com/gravitational/satellite/agent/proto/agentpb"
 	"github.com/gravitational/trace"
 )
 
-func checkManifest(env *localenv.LocalEnvironment, manifestPath, profileName string, autoFix bool) error {
+func checkManifest(env *localenv.LocalEnvironment, manifestPath, imagePath, profileName string, autoFix bool) error {
+	_, err := GetLocalClusterInfo(env)
+	if err != nil {
```

Comment:
Ignoring the error will make it difficult to understand why the command executes installer checks on an installed cluster if it has issues. Do we want to suggest running status in case of specific error conditions? (I don't really feel comfortable treating **any** error as a cluster not being there condition).

Reply:
Yeah, that's a bit tough. I feel like we don't have a bulletproof way to determine whether we have a cluster locally or now. For now, I've opted for the same heuristic as "gravity status" uses which is trying to query the planet agent status.